openzwavemqtt/const.py

```diff
@@ -69,8 +70,92 @@ class ValueIndex(IntEnum):
     METER_POWER = 2
     METER_RESET = 257
     # SensorMultilevel
-    SENSOR_MULTILEVEL_TEMPERATURE = 1
```

Comment:
Is this replaced with some other value now?

Reply:
SENSOR_MULTILEVEL_AIR_TEMPERATURE